main.go

```diff
@@ -412,7 +429,43 @@ func getBigqueryMessageOptions(msg *descriptor.DescriptorProto) (*protos.BigQuer
 	}, nil
 }
 
+func getInputParameters(req *plugin.CodeGeneratorRequest) map[string]string {
+	parameters := req.GetParameter()
+	extractedParams := make(map[string]string)
+
+	kvPairs := strings.Split(parameters, ",")
+	for _, element := range kvPairs {
+		kv := strings.Split(element, "=")
+		if len(kv) == 2 {
+			extractedParams[kv[0]] = kv[1]
+		}
+	}
+	return extractedParams
+}
+
+func getPolicyTags(req *plugin.CodeGeneratorRequest) (map[string]string, error) {
+	parameters := getInputParameters(req)
+
+	if val, ok := parameters["policy_tags_config_path"]; ok {
+		file, err := os.Open(val)
+		if err != nil {
+			return nil, fmt.Errorf("failed reading policy tags config file, %q", err)
+		}
+		myEnv, err := godotenv.Parse(file)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse policy tags config file, %q", err)
+		}
+		return myEnv, nil
```

Comment:
Little unclear of the purpose of using a `.env` file here? Is this some way of restricting the allowed policy tags that can be used?

Reply:
I just wanted something simple that does key/value pairs. I'm going to remove the entire config file element to this.

Reply:
Yeah reading Michal's comment it seems like a good idea to not have the dev/prod separation here. Good work on this though, adding lots of QoL to this tool :)